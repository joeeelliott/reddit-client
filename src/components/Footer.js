import React from 'react';

const Footer = () => {
  return (
    <div data-testid="footer" className="footer">
      <h1>Joe Elliott</h1>
      <p>&copy;</p>
    </div>
  )
}

export default Footer;