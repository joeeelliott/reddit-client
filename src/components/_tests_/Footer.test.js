import Footer from '../Footer';
import { shallow } from 'enzyme';

describe('Footer Component', () => {

  let component; 
  beforeEach(() => {
    component = shallow(<Footer />)
  });

  it('renders in as expected; matches snapshot', () => {
    expect(component).toMatchSnapshot(); 
  });
});