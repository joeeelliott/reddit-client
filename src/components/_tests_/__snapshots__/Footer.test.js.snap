// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders in as expected; matches snapshot 1`] = `ShallowWrapper {}`;
